### PR TITLE
Harsha | GALA Token Analysis

### PR DESCRIPTION
**Pull request by harshaharod21**
@chirag9127 
@encrypted-soul 

Link for deployed web app:  https://galatokenanalysis21.streamlit.app/

Team_lead: Harsha Harod
Team_member: Kavya Agrawal

### CODE REVIEW

**Comment by kavyagl2** [comment]
@1-Harshit Can you help me with this issue, actually I and Harsha are quite new to this thing and we have never used Github+Streamlit.

File: dec_2023/Team_Leader_Harsha_Harod/Gala_Token_Analysis.py [hunk 1]
```diff
@@ -0,0 +1,671 @@
...
+import streamlit as st
+from PIL import Image
+import pandas as pd
+
+
+
+def overview_page():
+
+    
+    img=Image.open("gala_logo.jpg")
```
Comment (1-Harshit) [inline dec_2023/Team_Leader_Harsha_Harod/Gala_Token_Analysis.py:11]: You have your image file `gala_logo.jpg` in `Assets/` directory

And we run the application from where `Gala_Token_Analysis.py`, so the path given should be `Assets/gala_logo.jpg` instead.

```suggestion
    img=Image.open("Assets/gala_logo.jpg")
```

Comment (harshaharod21) [reply dec_2023/Team_Leader_Harsha_Harod/Gala_Token_Analysis.py:11]: Got it ,will correct it,actually when I runned the code using "view in codespace",I again hade to change the path to "workspace/...../assets/...."

Comment (harshaharod21) [reply dec_2023/Team_Leader_Harsha_Harod/Gala_Token_Analysis.py]: I have updated sir,please do check.Thankyou!

**Pull request merged**